src/posthog-core.js

```diff
@@ -135,6 +137,24 @@ var create_mplib = function (token, config, name) {
     instance['people'] = new PostHogPeople()
     instance['people']._init(instance)
 
+    // if any instance on the page has debug = true, we set the
+    // global debug to be true
+    Config.DEBUG = Config.DEBUG || instance.get_config('debug')
+
+    // if target is not defined, we called init after the lib already
+    // loaded, so there won't be an array of things to execute
+    if (!_.isUndefined(target) && _.isArray(target)) {
+        // Crunch through the people queue first - we queue this data up &
+        // flush on identify, so it's better to do all these operations first
+        instance._execute_array.call(instance['people'], target['people'])
+        instance._execute_array(target)
+    }
+
+    if (instance.get_config('advanced_disable_decide')) {
+        // When /decide endpoint is disabled, nonone of the features below are available.
+        return instance
```

Comment:
This is a breaking change. We advertise `posthog.featureFlags` in our documentation, this will make that code throw if a flag is on. 

Suggestion: just gate the decide call.

Reply:
Agreed, per discussion on Slack